bc_obps/registration/api/user_operator.py

```diff
@@ -129,28 +130,18 @@ def save_operator(payload: UserOperatorOperatorIn, operator_instance: Operator,
         user_operator, created = UserOperator.objects.get_or_create(
             user=user,
             operator=created_or_updated_operator_instance,
-            role=UserOperator.Roles.ADMIN,
```

Comment:
This was causing the `get_or_create` to create a second record if an `industry_user` with a `reporter` role such as `bc-gov-cas-secondary` edited the form. That would cause an error when attempting to GET as it would return two records.

We are only setting the role now `if created:`